src/components/navBar/index.tsx

```diff
@@ -78,41 +78,42 @@ const NavBar = () => {
                 </ul>
             </div>
             <div className={styles.userProfile}>
-                {!isLoggedIn ? (
-                    <Link href={LOGIN_URL}>
-                        <button className={styles.signInLink}>
-                            Sign In With Github
+                {userData !== undefined &&
```

Comment:
instead of checking equality with undefined, we can just use `userData` in this line, this will return a falsy value. If you want to be more specific about the boolean then you can use it with `!!userData` which will convert the Falsy value to `false`

Reply:
Updated as per the suggestion 👍🏻